api/src/shared/application/scripts/base-script.js

```diff
@@ -0,0 +1,61 @@
+import 'dotenv/config';
+
+import Joi from 'joi';
+
+import { logger as defaultLogger } from '../../infrastructure/utils/logger.js';
+
+const META_INFO_SCHEMA = Joi.object({
```

Comment:
Je suggère plutôt `METAINFO_SCHEMA`. D'ailleurs dans le constructeur l'argument est nommé `metaInfo`, sans underscore.

```suggestion
const METAINFO_SCHEMA = Joi.object({
```

Reply:
Dans le constructeur on constate bien que ce sont deux mots en camel case (avec la majuscule à Info), donc si on respecte ça, en snake case, on écrira bien `META_INFO_SCHEMA`.